### DISCUSSION TITLE
Allow manually sorting columns from side panel in create table and update table modes

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#16843] [Contribute to Supabase]
We should be able to reorder columns during the edit table and create table stage in the side panel. Currently, we can't change the order on existing tables. Ideally, we'd be able to sort columns from both.

![CleanShot 2023-08-14 at 10 54 37@2x](https://github.com/supabase/supabase/assets/105593/727a6a49-3f23-4f41-a3f6-2a08e6c81a9d)

**Comment by faizanraso** [comment]
Hi @saltcod I've submitted a PR to fix this issue - https://github.com/supabase/supabase/pull/16512

**Comment by Nikhil743** [comment]
Hey, @saltcod would like to work on this issue.
could you assign it to me?

**Comment by Gingaaa** [comment]
Hey I have worked on around 7-8 company based project, but never countribute to open source, this is my first time can you assign me this task.

**Comment by JediPixels** [comment]
I also would love to see this happen... reorder fields after the table is created.

**Reply by GaryAustin1** [reply]
Not sure what the target here was as you can't easily reorder columns in Postgres if you have existing data.  You have to add new columns (which go at end of table) copy data to them and delete the old columns.
